### ISSUE TITLE
about meta-learning

### ISSUE THREAD

**Issue by btwewek** [automl/auto-sklearn#606]
I'm new to autoML, and I have read paper of auto-sklearn,
but I am not sure about my understanding of meta-learning in auto-sklearn.
Suppose that we have only 3 ML algorithm, 
The meta-learning will suggest one ML algorithm,
OR, meta-learning will give us ensemble model(include 3 ML algorithm) and roughly hyper-parameter setting of each ML algorithm.
Then we can use some optimization method to optimize hyper-parameter of ML algorithm.

In short, I am not sure what is the output of meta-learning.
If meat-learning can suggest one ML algorithm, I can fine-tune that algorithm by myself.

**Comment by mfeurer** [comment]
Please check [this publication](https://ml.informatik.uni-freiburg.de/papers/15-AAAI-MI-SMBO.pdf) for more details. In short, the outcome is a list of ML algorithms and their hyperparameter settings, which can be applied one after the other. Therefore, if you have a single ML algorithm with a real-valued hyperparameter, the meta-learning algorithm can (in theory) recommend and infinite amount of algorithm-hyperparameter combinations. 

I hope this helps.
